src/Logging/Logging/LogMessageFormatter.cs

```diff
@@ -22,11 +30,16 @@ public class LogMessageFormatter
             return builder.ToString();
         }
 
-        protected virtual string FormatLogLevel(
-            LogLevel logLevel) => logLevel.ToString();
+        protected virtual string FormatLogLevel(LogLevel logLevel)
+        {
+            return logLevel.ToString().ToUpper();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)